src/wallet/rpcwallet.cpp

```diff
@@ -2550,12 +2463,7 @@ static UniValue listwallets(const JSONRPCRequest& request)
     UniValue obj(UniValue::VARR);
 
     for (const std::shared_ptr<CWallet>& wallet : GetWallets()) {
-        if (!EnsureWalletIsAvailable(wallet.get(), request.fHelp)) {
-            return NullUniValue;
-        }
```

Comment:
Note: PR is just dropping the Ensure check here which has been around since 9508761ed69a30f4af24fbc9274176056431abfb. It seems like it never did anything because the wallet pointer would never be null, and it was probably added originally as a precaution

Reply:
Agree.